recipes/napari-unicell/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "napari-unicell" %}
+{% set version = "0.0.1.post3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-unicell-{{ version }}.tar.gz
+  sha256: 4492664fef7f7ca31afee644e1e6a53eaab97ea78af148b76faaa36f78b46095
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pytorch
+    - imagecodecs
+    - scipy
+    - numpy
+    - magicgui
+    - qtpy
+    - scikit-image
+    - monai
+    - einops
+    - pyqt
+    - napari
+    - napari-plugin-engine
+
+test:
+  imports:
+    - napari_unicell
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/JunMa11/napari-unicell
```

Comment:
This is giving a 404, so at the very least this should be updated. Do you still want this recipe added?

Reply:
Seems to be back online!